### PR TITLE
some progress on how to play

### PR DESCRIPTION
**Pull request by charles-uno**
Not as much as I hoped to get done over the weekend, but at least a bit of progress!

### CODE REVIEW

File: _chapters/00-outline.md [hunk 1]
```diff
@@ -19,6 +19,13 @@ Players get to decide what sort of story they're telling, and what details matte
 Everything else is allowed to slide.
 The result is a punchy, quick-moving game with plenty of opportunities for memorable moments.
 
+---
+
+The players and the GM work together to tell stories about the adventures of the PCs.
+They could tell an action story, where the PCs hack away at bloodthirsty monsters in defense of civilization.
```
Comment (agude) [inline _chapters/00-outline.md:25]:
```suggestion
They may tell an action story, where the PCs hack away at bloodthirsty monsters in defense of civilization.
```

File: _chapters/00-outline.md [hunk 2]
```diff
@@ -196,12 +203,43 @@ Build them just like a normal NPC with aspects; the only difference is how you n
 
 # How to Play
 
-## Setting the Scene
+The story in {GameName} plays out scene-by-scene.
+The GM generally opens each scene with a quick description.
+Play proceeds conversationally from there as the players and GM improvise back and forth.
+Players speak as their PCs and describe their actions, while the GM illustrates how the world responds.
```
Comment (agude) [inline _chapters/00-outline.md:209]: > illustrates 

Makes me think of drawing a picture... :-/

File: _chapters/00-outline.md [hunk 3]
```diff
@@ -196,12 +203,43 @@ Build them just like a normal NPC with aspects; the only difference is how you n
...
+Players may also ask the GM for further details to help them immerse themselves in the PCs' surroundings.
+
+Play is punctuated by die rolls, which mediate the divide between player and PC.
```
Comment (agude) [inline _chapters/00-outline.md:212]: > mediate

Don't love it...

Here are some GPT suggestions:

1. Die rolls serve as a means of determining the outcomes of player and PC actions.
2. The game's mechanics are driven by die rolls, which act as a go-between for player and PC decisions.
3. Rolling the dice creates a sense of uncertainty and chance, bridging the gap between player and PC agency.
4. Die rolls provide a framework for players to interact with their characters and make in-game decisions.
5. The use of dice adds an element of randomness and luck to the gameplay, mediating the relationship between player and PC.

File: _chapters/00-outline.md (hunk 2 above)
Comment (agude) [reply _chapters/00-outline.md:209]: Some options from GPT:

1. The GM narrates the world's reactions to the PCs' actions.
2. The GM describes the consequences of the PCs' actions.
3. The GM shapes the world in response to the PCs' choices.
4. The GM reveals how the world reacts to the PCs' decisions.
5. The GM guides the narrative based on the PCs' actions and the world's response.

File: _chapters/00-outline.md [hunk 4]
```diff
@@ -196,12 +203,43 @@ Build them just like a normal NPC with aspects; the only difference is how you n
...
+PC choices are made by the players, while the success of PC actions is judged by dice.
```
Comment (agude) [inline _chapters/00-outline.md:213]:
```suggestion
PC choices are made by the players, while the success of PC actions is judged by the dice.
```

File: _chapters/00-outline.md [hunk 5]
```diff
@@ -196,12 +203,43 @@ Build them just like a normal NPC with aspects; the only difference is how you n
...
+That is, the player decides which NPCs to attack and which to barter with.
+The dice show whether a sword strikes home or a lie rings true.
+This dichotomy gives each PC quantitative strengths and weaknesses while still allowing players the freedom to play their PCs however they like.
```
Comment (agude) [inline _chapters/00-outline.md:216]: Maybe:

> This dichotomy gives each PC quantitative strengths and weaknesses, while still allowing players the freedom to play their PCs in a way that feels true to them.

File: _chapters/00-outline.md [hunk 6]
```diff
@@ -196,12 +203,43 @@ Build them just like a normal NPC with aspects; the only difference is how you n
...
 
 ## Rolling the Dice
 
+The GM may prompt players to roll at any time.
+
+Sometimes you roll because your PC is attempting to do something.
+- Naka throwing a fireball
+- TBY bluffing at cards
+
+Sometimes the GM prompts you to roll unexpectedly.
```
Comment (agude) [inline _chapters/00-outline.md:226]: We need to include something about "Don't role 'spot' and then go 'I guess you don't see anything'", failure of a roll leads to immediate consequences.

File: _chapters/00-outline.md [hunk 7]
```diff
@@ -196,12 +203,43 @@ Build them just like a normal NPC with aspects; the only difference is how you n
...
+- It's ok to suspend disbelief for the sake of theatricality.
+- Example of difficult but boring stuff...
+- Example of easy stuff where failure is interesting...
+
 ## Fate Points
 
+TBY's trouble tells us that he plays cards.
+Maybe he's bad, and a degenerate gambler?
+Maybe he's great, but got cheated?
+Tagging the aspect allows us to flesh out that part of the story.
```
Comment (agude) [inline _chapters/00-outline.md:241]: Love it

**Review by agude — APPROVED** [review]
Some minor comments.

File: _chapters/00-outline.md (hunk 5 above)
Comment (charles-uno) [reply _chapters/00-outline.md:216]: ooh I like this

**Pull request merged**